ingestion_server/ingestion_server/elasticsearch_models.py

```diff
@@ -280,6 +269,19 @@ def get_aspect_ratio(height, width):
             aspect_ratio = Image.AspectRatios.SQUARE.name
         return aspect_ratio.lower()
 
+    @staticmethod
+    def get_extension(url):
+        """
+        Get the extension from the last segment of the URL separated by a dot.
+        TODO: Use the `filetype` field once the following issue is completed:
+        https://github.com/WordPress/openverse-catalog/issues/536
+        """
+        extension = url.split(".")[-1].lower()
```

Comment:
I think 
```suggestion
        if filetype:
            return filetype
        extension = url.split(".")[-1].lower()
```

Reply:
I thought I had answered this, in my mind I wrote that I only focused on changes needed for Audio here 😅